helm/scylla-manager/values.yaml

```diff
@@ -7,7 +7,7 @@ fullnameOverride: ""
 image:
   repository: scylladb
   pullPolicy: IfNotPresent
-  tag: 2.2.4
+  tag: 2.3.0
```

Comment:
This should be reflected in our deploy folder but isn't. 
Looks like there's a not needed version override in `helm/deploy/manager_prod.yaml`.

Reply:
removed the override